### PR TITLE
Initial changes to onboard WOPI documentation for android

### PR DESCRIPTION
**Pull request by jj-invincible**
I'm sending this just as a sample. Do not approve this to master. We need to make more changes to it. I wanted to get feedback and understand whether we are doing the changes correctly

### CODE REVIEW

File: docs/native/src/index.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 ..  _native wopi:
 
-Using the WOPI protocol to integrate with Office for iOS
-========================================================
+Using the WOPI protocol to integrate with Office for iOS and Android
```
Comment (tylerbutler) [inline docs/native/src/index.rst]: You can actually use the `|Office iOS Android|` placeholder in headings as well.

File: docs/_shared/rst_prolog.rst [hunk 2]
```diff
@@ -23,6 +23,10 @@
 
 ..  |Office iOS| replace:: Office for iOS
 
+..  |Office iOS Android| replace:: Office for iOS and Android
```
Comment (tylerbutler) [inline docs/_shared/rst_prolog.rst]: How about `Office Mobile` for this placeholder?

File: docs/native/src/index.rst (hunk 1 above)
Comment (jj-invincible) [reply docs/native/src/index.rst]: Will update

File: docs/_shared/rst_prolog.rst (hunk 2 above)
Comment (jj-invincible) [reply docs/_shared/rst_prolog.rst]: Well, we have windows phone too.

**Comment by jj-invincible** [comment]
Closing the old one.

**Pull request closed without merging**